opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -191,7 +191,7 @@ namespace Opm
 
                 // No per cell data is written for initial step, but will be
                 // for subsequent steps, when we have started simulating
-                output_writer_.writeTimeStepWithoutCellProperties( timer, state, well_state );
+                output_writer_.writeTimeStepWithoutCellProperties( timer, state, well_state, {} );
```

Comment:
The `extra_data` argument is optional (for good or for bad) - so it should not be required to pass a `{}` argument.